src/main/java/com/sourcegraph/javagraph/Project.java

```diff
@@ -12,7 +12,12 @@
     /**
      * Default source code version (language level)
      */
-    public static final String DEFAULT_SOURCE_CODE_VERSION = "1.8";
+    String DEFAULT_SOURCE_CODE_VERSION = "1.8";
+
+    /**
+     * Project initialization phase
```

Comment:
Describe what sort of things should be done in this phase. When should a subclass implement it?